androidApp/src/main/res/values/strings.xml

```diff
@@ -18,21 +27,31 @@
     <string name="more_section_support">General MBTA Information &amp; Support</string>
     <string name="more_section_support_note">Monday through Friday: 6:30 AM - 8 PM</string>
     <string name="more_title">MBTA Go</string>
+    <string name="nearby_transit">Nearby Transit</string>
     <string name="nearby_transit_link">Nearby</string>
+    <string name="no_predictions">Predictions unavailable</string>
     <string name="no_service">No Service</string>
+    <string name="no_service_today">No service today</string>
     <string name="no_stops_nearby">You’re outside the MBTA service area.</string>
     <string name="no_stops_nearby_title">No nearby stops</string>
+    <string name="northbound">Northbound</string>
     <string name="now">Now</string>
     <string name="other_link_privacy_policy">Privacy Policy</string>
     <string name="other_link_source_code">View source on GitHub</string>
     <string name="other_link_tos">Terms of Use</string>
+    <string name="outbound">Outbound</string>
+    <string name="pin_route">Star route</string>
+    <string name="recenter" tools:ignore="MissingTranslation">Recenter</string>
```

Comment:
The map recenter button has no alt text on iOS, and I don't think it's correct to add it in this PR, so I'm punting on these translations for now here too.